h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstCumMax.java

```diff
@@ -3,9 +3,7 @@
 
 public class AstCumMax extends AstCumu {
   @Override
-  public int nargs() {
-    return 1 + 1;
-  }
+  public int nargs() { return 1 + 2; }
```

Comment:
I see, its not needed, because all of the child classes of AstCumu (cumsum, cumprod, etc) override this properly